app/api/uploadthing/core.ts

```
@@ -9,122 +9,122 @@ const f = createUploadthing();
+
+			// Return userId to be used in onUploadComplete
+			return { userId: user.id };
+			// This code runs on your server before upload
+			// const user = await auth(req);
+
+			// If you throw, the user will not be able to upload
+			// if (!user) throw new UploadThingError("Unauthorized");
+
+			// Whatever is returned here is accessible in onUploadComplete as `metadata`
+			// return { userId: user.id };
+			return {};
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)